Jellyfin.Data/Enums/CollectionType.cs

```diff
@@ -0,0 +1,147 @@
+namespace Jellyfin.Data.Enums;
+
+/// <summary>
+/// Collection type.
+/// </summary>
+public enum CollectionType
+{
+    /// <summary>
+    /// Unknown collection.
+    /// </summary>
+    Unknown = 0,
+
+    /// <summary>
+    /// Movies collection.
+    /// </summary>
+    Movies = 1,
+
+    /// <summary>
+    /// Tv shows collection.
+    /// </summary>
+    TvShows = 2,
+
+    /// <summary>
+    /// Music collection.
+    /// </summary>
+    Music = 3,
+
+    /// <summary>
+    /// Music videos collection.
+    /// </summary>
+    MusicVideos = 4,
+
+    /// <summary>
+    /// Trailers collection.
+    /// </summary>
+    Trailers = 5,
+
+    /// <summary>
+    /// Home videos collection.
+    /// </summary>
+    HomeVideos = 6,
+
+    /// <summary>
+    /// Box sets collection.
+    /// </summary>
+    BoxSets = 7,
+
+    /// <summary>
+    /// Books collection.
+    /// </summary>
+    Books = 8,
+
+    /// <summary>
+    /// Photos collection.
+    /// </summary>
+    Photos = 9,
+
+    /// <summary>
+    /// Live tv collection.
+    /// </summary>
+    LiveTv = 10,
+
+    /// <summary>
+    /// Playlists collection.
+    /// </summary>
+    Playlists = 11,
+
+    /// <summary>
+    /// Folders collection.
+    /// </summary>
+    Folders = 12,
+
+    /// <summary>
+    /// Tv show series collection.
+    /// </summary>
+    TvShowSeries = 101,
+
+    /// <summary>
+    /// Tv genres collection.
+    /// </summary>
+    TvGenres = 102,
+
+    /// <summary>
+    /// Tv genre collection.
+    /// </summary>
+    TvGenre = 103,
```

Comment:
What's the difference between these two?

Reply:
Yay legacy code where they added support for both…

Reply:
Can't we write a migration and get rid of them?

Reply:
We could mark one as `[Obsolete]` or something right?

Reply:
The difference is actually as follows so it needs to stay... I've opted to just hide them from the api spec

https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Controller/Entities/UserViewBuilder.cs#L73:L77

Reply:
Aah so one is really a TvGenreList (list of TVGenres) and the other one a list of items in a specific genre.